src/main/kotlin/racingcar/model/Race.kt

```diff
@@ -0,0 +1,34 @@
+package racingcar.model
+
+import camp.nextstep.edu.missionutils.Randoms
+
+class Race(private val users: List<User>) {
+
+    fun play() {
+        users.forEach { user ->
+            if (getRandomNum() >= MIN_MOVE_NUMBER) {
+                user.move()
+            }
+        }
+    }
+
+    private fun getRandomNum(): Int = Randoms.pickNumberInRange(RANDOM_MIN_NUMBER, RANDOM_MAX_NUMBER)
+
+    fun getScores(): Map<String, Int> {
+        return users.associate { it.displayName() to it.getScore() }
+    }
+
+    fun getWinners(): List<String> {
+        val maxScore = users.maxOf { it.getScore() }
+        return users
+            .filter { it.getScore() == maxScore }
+            .map { it.displayName() }
+    }
+
+    companion object {
+        private const val MIN_MOVE_NUMBER = 4
```

Comment:
```suggestion
        private const val MOVE_THRESHOLD = 4
```
MIN_MOVE_NUMBER라고 하면 RANDOM_MIN_NUMBER와의 의미가 혼동될 수 있으니 움직임을 위한 기준값이라는 의미가 변수명에 포함되면 더 좋을 수도 있을 것 같아요.

Reply:
이름 정하는게 제일 어려운 것 같습니다...
THRESHOLD라는 명칭을 사용하면 확실히 의도가 명확해질 것 같네요
감사합니다!